### PR TITLE
fix creation transaction rpc

### PR DESCRIPTION
**Pull request by winsvega**
an issue occured when `to` field was set in json as empty string

### CODE REVIEW

File: libweb3jsonrpc/JsonHelper.cpp [hunk 1]
```diff
@@ -360,8 +360,8 @@ TransactionSkeleton toTransactionSkeleton(Json::Value const& _json)
 
 	if (!_json["from"].empty())
 		ret.from = jsToAddress(_json["from"].asString());
-	if (!_json["to"].empty() && _json["to"].asString() != "0x")
-		ret.to = jsToAddress(_json["to"].asString());
+    if (!_json["to"].empty() && _json["to"].asString() != "0x" && !_json["to"].asString().empty())
```
Comment (gumb0) [inline libweb3jsonrpc/JsonHelper.cpp]: An empty string in `to` field sounds to me like an invalid json for transaction Why do you want to ignore this error and create and invalid `TransactionSkeleton` ?

Comment (winsvega) [reply libweb3jsonrpc/JsonHelper.cpp]: but `"to" : "0x"`  is not considered to be invalid? 

hm. in tests it is empty `to` string that indicates creation.

Comment (gumb0) [reply libweb3jsonrpc/JsonHelper.cpp]: Ah yes, right, sorry empty recipient means contract creation

**Comment by winsvega** [comment]
converting all tabs will change the whole file. are you sure?
changed lines are already clang-formated.

**Comment by gumb0** [comment]
I am sure, we do this for all touched files, so that none of the files have a mixture of tabs and spaces

**Comment by gumb0** [comment]
ah now you reformatted all of it.
Could you reformat only the changed lines and for the rest of the file just convert tabs?

**Comment by winsvega** [comment]
why just tabs? whats wrong with converting the whole file? 
I thought if we are converting the file then its good to format all the file. otherwise why do we use clang?

**Comment by gumb0** [comment]
Reformatting the whole file would mess up with `git blame` results (that is, it would be difficult to find out who was the author of particular lines and when)
While the whitespace only can be ignored by the option of `git blame`

**Pull request merged**